src/Umbraco.Web.UI.Client/src/views/components/blockcard/umbBlockCard.component.js

```diff
@@ -14,9 +14,39 @@
             }
         });
 
-    function BlockCardController() {
+    function BlockCardController($scope, umbRequestHelper) {
 
         var vm = this;
+        vm.styleBackgroundImage = "transparent";
+
+        var unwatch = $scope.$watch("vm.blockConfigModel.thumbnail", (newValue, oldValue) => {
+            console.log("updateThumbnail")
+            if(newValue !== oldValue) {
+                vm.updateThumbnail();
+            }
+        });
+
+        vm.$onInit = function () {
+
+            vm.updateThumbnail();
+
+        }
+        vm.$onDestroy = function () {
+            unwatch();
+        }
+
+        vm.updateThumbnail = function () {
+            if (vm.blockConfigModel.thumbnail == null || vm.blockConfigModel.thumbnail === "") {
+                vm.styleBackgroundImage = "transparent";
```

Comment:
@nielslyngsoe notice `transparent` isn't a valid value for `background-image` but `none` is.

Reply:
@bjarnef Thanks, what should I have done without you :-D Thanks a lot